phpBB/phpbb/cache/driver/null.php

```diff
@@ -15,137 +15,8 @@
 
 /**
 * ACM Null Caching
+* @deprecated 3.1.5 (to be removed: 3.3.0)
```

Comment:
As I mentioned on IRC earlier, this doesn't break anything at all. The null driver will still work as it extends the dummy driver.